### PR TITLE
[refs #00209] Heading line heights

### PR DESCRIPTION
**Pull request by penpadstudio**
Updated Headings to correct sizing and added specific line heights. Also added missing H6. Ready to be merged into develop branch on code approval.

### CODE REVIEW

**Review by cehwitham — COMMENTED** [review]
Is  there a need to introduce all the line heights @Pro-Paul ? We didn't have this currently?

**Comment by penpadstudio** [comment]
We didn't have the line heights set, they were all generic so they have now been set, I will be doing the same for body copy as they also have not been set according to prior design format.

**Review by cehwitham — CHANGES_REQUESTED** [review]
Do we not already have a relative method for handling line heights? We shouldn't really be using exact values for line heights as it's not best practice.

https://csswizardry.com/2012/11/code-smells-in-css/#hard-codedabsolute-values

**Comment by penpadstudio** [comment]
I might need some help to talk through what needs to be updated and how in the code for this one @cehwitham

**Comment by cehwitham** [comment]
Probably one to visit with @csswizardry

File: tools/_tools.mixins.scss [hunk 1]
```diff
@@ -51,13 +51,22 @@
   font-size: $font-size;
   font-size: ($font-size / $global-font-size) * 1rem;
 
+  // If we haven’t specified a line-height explicitly, let’s automatically
+  // create one that sits on the next closest unit of our baseline grid.
   @if ($line-height == auto) {
     line-height: ceil($font-size / $global-line-height) * ($global-line-height / $font-size);
```
Comment (cehwitham) [inline tools/_tools.mixins.scss:57]: Won't this always = 1? Should one of the variables be a different font size? Global or something?

Comment (csswizardry) [reply tools/_tools.mixins.scss:57]: The `ceil()` rounds up the first equation so it will always be >=1 👍

Comment (cehwitham) [reply tools/_tools.mixins.scss:57]: Ah I'd missed the middle set of brackets.

**Review by csswizardry — APPROVED** [review]
Worked on this with @Pro-Paul; we’re both happy with it.

**Review by cehwitham — APPROVED** [review]

**Pull request merged**